lib/engine/config/game/g_1835.rb

```diff
@@ -0,0 +1,583 @@
+      "logo": "1835/ORR",
+      "tokens": [
+        0,
+        40,
+        100
+      ],
+      "color": "orange"
+    },
+    {
+      "sym": "PRR",
+      "name": "Pink Railroad",
+      "logo": "1835/PRR",
+      "tokens": [
+        0,
+        40,
+        100
+      ],
+      "color": "pink"
+    },
+    {
+      "sym": "VRR",
+      "name": "Violet Railroad",
+      "logo": "1835/VRR",
+      "tokens": [
+        0,
+        40,
+        100
+      ],
+      "color": "violet"
+    },
+    {
+      "sym": "RRR",
+      "name": "Red Railroad",
+      "logo": "1835/RRR",
+      "tokens": [
+        0,
+        40,
+        100
+      ],
+      "color": "red"
+    },
+    {
+      "sym": "LBRR",
+      "name": "Light Brown Railroad",
+      "logo": "1835/LBRR",
+      "tokens": [
+        0,
+        40,
+        100
+      ],
+      "color": "lightBrown"
+    },
+    {
+      "sym": "TRR",
+      "name": "Turquoise Railroad",
+      "logo": "1835/TRR",
+      "tokens": [
+        0,
+        40,
+        100
+      ],
+      "color": "turquoise"
+    },
+    {
+      "sym": "WRR",
+      "name": "White Railroad",
+      "logo": "1835/WRR",
+      "tokens": [
+        0,
+        40,
+        100
+      ],
+      "color": "white"
+    },
+    {
+      "sym": "YRR",
+      "name": "Yellow Railroad",
+      "logo": "1835/YRR",
+      "tokens": [
+        0,
+        40,
+        100
+      ],
+      "color": "yellow"
+    }
+  ],
+  "trains": [
+    {
+      "name": "2",
+      "distance": 2,
+      "price": 80,
+      "rusts_on": "4",
+      "num": 9
+    },
+    {
+      "name": "2+2",
+      "distance": 2,
+      "price": 120,
+      "rusts_on": "4+4",
+      "num": 4
+    },
+    {
+      "name": "3",
+      "distance": 3,
+      "price": 180,
+      "rusts_on": "6",
+      "num": 4
+    },
+    {
+      "name": "3+3",
+      "distance": 3,
+      "price": 270,
+      "rusts_on": "6+6",
+      "num": 3
+    },
+    {
+      "name": "4",
+      "distance": 4,
+      "price": 360,
+      "num": 3
+    },
+    {
+      "name": "4+4",
+      "distance": 4,
+      "price": 440,
+      "num": 1
+    },
+    {
+      "name": "5",
+      "distance": 5,
+      "price": 500,
+      "num": 2
+    },
+    {
+      "name": "5+5",
+      "distance": 5,
+      "price": 600,
+      "num": 1
+    },
+    {
+      "name": "6",
+      "distance": 6,
+      "price": 600,
+      "num": 2
+    },
+    {
+      "name": "6+6",
+      "distance": 6,
+      "price": 720,
+      "num": 4
+    }
+  ],
+  "hexes": {
+    "white": {
+      "": [
+        "A13",
+        "B10",
+        "B12",
+        "B14"
+      ],
+      "city=revenue:0": [
+        "A11"
+      ]
+    },
+    "yellow": {
+      "city=revenue:20;path=a:2,b:_0": [
+        "C13"
+      ]
+    }
+  },
+
+  "phases": [
+    {
+      "name": "1.1",
+      "train_limit": 4,
+      "tiles": [
+        "yellow"
+      ],
+      "operating_rounds": 1
+    },
+    {
+      "name": "1.1",
```

Comment:
Use unique names for the phases.
I haven't played 35 in over 25 years so I do not remember any details.
If phase is triggered by train buy you might need 'on'.

Reply:
All the phase stuff and companies are placeholders that I carved out of the original template that are left because they weren't causing problems. THere' sa list at the top, including these, of what still needs to be done. They're placeholders becuase I need them to work on the map.